framework/materials/material.h

```diff
@@ -17,7 +17,8 @@ enum class OperationType
   FROM_ARRAY = 1,
   SIMPLE_ONE_GROUP = 20,
   EXISTING = 22,
-  OPENSN_XSFILE = 23
+  OPENSN_XSFILE = 23,
+  OPENMC_XSLIB = 24
```

Comment:
just be aware that users may wonder why one is XSFILE and the other is XSLIB. I get the file for ascii text and the lib for h5 database but users may not get this and may wonder why opensn is a file and openmc is a lib. Jus for you to consider and decide whether using the same word is better or not

Reply:
I think it's probably best to leave it as XSLIB. It is possible to create an h5 file from OpenMC that contains xs info that is not the same as a library.